source/numerics.tex

```diff
@@ -8023,10 +8023,10 @@
 the
 \tcode{valarray}
 template, which is one-dimensional.
-The set of one-dimensional index values specified by a
-\tcode{gslice}
-are $$k = s + \sum_ji_jd_j$$
-% \$k = s + sum from j \{ i sub j d sub j \}\$
+The set of one-dimensional index values specified by a \tcode{gslice} are
+
```

Comment:
Oh, I just tried to make it consistent with the other one, which is also an equation in the middle of a sentence, and already had the empty lines around it. If empty lines around \[...\] introduce paragraph breaks that are unwanted in those cases, I guess the empty lines should be removed there, as well?